CHANGELOG.md

```diff
@@ -2,6 +2,100 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.18 - April 22, 2021
+## Breaking changes ⚠️
+* ResourceOptions contains tile store instance (TileStore API). Tile store usage is enabled by default, 'ResourceOptions.tileStoreEnabled' flag is introduced to disable it.
+* Rename MapView plugin extension functions. ([#272](https://github.com/mapbox/mapbox-maps-android/pull/272))
+  - mapView.getAnnotationPlugin() -> mapView.annotations
+  - mapView.getGesturesPlugin() -> mapView.gestures
+  - mapView.getOverlayPlugin() -> mapView.overlay() // using function here because of experimental annotation
+  - mapView.getLocationComponentPlugin() -> mapView.location
+  - mapView.getCameraAnimationsPlugin() -> mapView.camera
+  - mapView.getAttributionPlugin() -> mapView.attribution
+  - mapView.getCompassPlugin() -> mapView.compass
+  - mapView.getLogoPlugin() -> mapView.logo
+  - mapView.getScaleBarPlugin() -> mapView.scalebar
+* Remove deprecated location plugin ([#276](https://github.com/mapbox/mapbox-maps-android/pull/276))
+* Add feature sdk initialisation ([#269](https://github.com/mapbox/mapbox-maps-android/pull/269))
+  - Load the Mapbox Street style by default if user doesn't load any style before the onStart lifecycle event.
+  - Introduce `CredentialsManager` to manage mapbox access token, when all `MapView`s should use same token could be handled by using `CredentialsManager.shared` static object.
+  - Introduce `MapInitOptions` to replace MapboxMapOptions. When adding a map programmatically, client codes can be written in a following way
+  ```
+   // set map options
+    val mapOptions = MapOptions.Builder()
+      .constrainMode(ConstrainMode.HEIGHT_ONLY)
+      .glyphsRasterizationOptions(
+        GlyphsRasterizationOptions.Builder()
+          .rasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
+          .build()
+      )
+      .build()
+
+    // set token and cache size for this particular map view
+    val resourceOptions = ResourceOptions.Builder()
+      .accessToken(getString(R.string.mapbox_access_token))
+      .cacheSize(75_000L)
+      .build()
+
+    // set initial camera position
+    val initialCameraOptions = CameraOptions.Builder()
+      .center(Point.fromLngLat(-122.4194, 37.7749))
+      .zoom(9.0)
+      .build()
+
+    val mapboxMapOptions =
+      MapInitOptions(this, resourceOptions, mapOptions, initialCameraOptions, true)
+    // create view programmatically
+    customMapView = MapView(this, mapboxMapOptions)
+  ```
```

Comment:
Should we leave this example code to the Github release notes and the new guide?

Reply:
I think it doesn't hurt to keep them here, we can reorganise it later if needed.